frontend/src/components/page/ContestTop/ContestTop.tsx

```diff
@@ -1,3 +1,78 @@
+import { useAccessTokenClaimValue } from "@/src/globalStates/credential";
+import { useGetContest, useRegisterMe, useRouterContestSlug } from "@/src/usecases/contest";
+import { Box, Button, Card, CardBody, CardHeader, Heading, useToast } from "@chakra-ui/react";
+import { useTimer } from "react-timer-hook";
+
+const TimerComponent = (props: { expiryTimestamp: Date}) => {
+  const timer = useTimer({ expiryTimestamp: props.expiryTimestamp });
+  // leading zero を追加する処理
+  let seq = [
```

Comment:
let の必要はないので const に修正お願いします:bow:
```suggestion
  const seq = [
```

配列の要素の変更は変数への再代入ではないので const でも動きます

Reply:
これ知りませんでした......直します